mocks/msw-server.js

```diff
@@ -13,35 +13,35 @@ import { matchRequestUrl } from 'msw';
 import { setupServer } from 'msw/node';
 import { handlers } from './auth/handlers';
 
-
 // This configures a request mocking server with the given request handlers.
 export const server = setupServer(...handlers);
 
 export function waitForRequest(method, url) {
-  let requestId = ''
+  let expectedRequestId = '';
 
   return new Promise((resolve, reject) => {
-    server.events.on('request:start', (req) => {
-      const matchesMethod = req.method.toLowerCase() === method.toLowerCase()
-      const matchesUrl = matchRequestUrl(req.url, url).matches
+    server.events.on('request:start', ({ request, requestId }) => {
```

Comment:
The call signature of the life-cycle events has changed a bit. I've adjusted it here.